elections/steering/documentation/template/README.md

```diff
@@ -114,7 +114,7 @@ Committee's governance will be considered for voter exception.
 Examples of contributions that would be considered:
 * Slack admins who are not active in GitHub
 * K8s Infra staff doing mostly support
-* Working Group leads without a lot of GitHub activity
+* Working Group or User Group leads without a lot of GitHub activity
```

Comment:
Just for (my) clarification. The source of truth for working group leads is here — https://github.com/kubernetes/community/blob/master/sigs.yaml

What do we mean here by `User Group` and where to find the information about leads of User Group?

Reply:
I am unsure myself - I put it in the template because it has been in the actually-used README for years, so I wanted the template to reflect the reality.

Reply:
In this case, should we remove the `User Group` part from the (source) README as well? to remove any ambiguity around it?

@jberkus, could you add context to the `User Group` part?

Reply:
User Groups were deprecated as a community group type. As such, this can be removed.

Reply:
For more context: We span down the remaining user group and the concept last year in https://github.com/kubernetes/community/pull/7407

Reply:
Per this discussion, I have removed the references to "User Group" from the current election as well as the directions/template.